.nvmrc

```diff
@@ -1 +1 @@
-14.3.0
+v14.18.1
```

Comment:
> updated node version specified in .nvmrc

can i ask why? this is not clear for me on first sight.

Reply:
`error jest-regex-util@27.0.6: The engine "node" is incompatible with this module. Expected version "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0". Got "14.3.0"`

`v14.18.1` is the latest `v14.x.x` lts version. there is no `v15.x.x` lts version and expo has not yet been testet with `v16.x.x` versions

Reply:
why with `v` now? and not just `14.18.1` as before? in .yvmrc we also add version number without `v`

Reply:
no particular reason. it came from me copying the name of the lts version, which includes the `v`. nvm acts the same way, regardless of the `v` being there or not